### PR TITLE
[patch] added webgo

### PR DESCRIPTION
**Pull request by bnkamalesh**
Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [ ] A `Dockerfile` exists, or can be built ?
+ [ ] I've updated `CI` config file (and **commited** `.travis.yml`)
~~~sh
bin/make ci_config
~~~
+ [ ] I've updated `.dependabot` config file  (and **commited** `.dependabot/config.yml`)
~~~sh
bin/make deps_config
~~~
+ [ ] All tests passes ?
~~~
export FRAMEWORK=<MY_FRAMEWORK>
shards install
shards build --release
bin/make config
bin/neph ${FRAMEWORK}
crystal spec
~~~

--------------
edit : Closes https://github.com/the-benchmarker/web-frameworks/issues/1916

### CODE REVIEW

File: go/webgo/go.sum [hunk 1]
```diff
@@ -0,0 +1,4 @@
+github.com/bnkamalesh/web-frameworks v1.2.6 h1:Y4/yaXLCchPsc5keLxBKYrDaGhxJkcLuMG9bbJcVNGk=
```
Comment (waghanza) [inline go/webgo/go.sum]: could you remove this file ?
no need in our case

File: go/webgo/go.mod [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module main
+
+go 1.13
```
Comment (waghanza) [inline go/webgo/go.mod]: could you remove it ? (I know it's created by `go`)

Comment (waghanza) [reply go/webgo/go.mod]: sorry, I'd better be more precise

only this line, not the whole file

Comment (bnkamalesh) [reply go/webgo/go.mod]: Oh, I've added the go.mod file back, and remove `go 1.13` from it.

**Pull request merged**